src/apiCalls.js

```diff
@@ -0,0 +1,13 @@
+const apiCalls = {
+  getMovieData(id) {
+    if (!id) {
+      return fetch("https://rancid-tomatillos.herokuapp.com/api/v2/movies");
+    } else {
+      return fetch(
+        `https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`
+      );
+    }
+  }
+};
+
```

Comment:
Looks great, love that this is dynamic. We will go back and add a checkResponse function.